### ISSUE TITLE
Demo Meeting March 17 2022 

### ISSUE THREAD

**Issue by ssbilkhu** [AAFC-BICoE/dina-planning#322]
Meeting notes from March 17th demo meeting:

•	Restricted Component on Material Samples
•	Improved Read Only View of Material Samples (Restrictions, tags, and Alerts)
•	Allow searching on more than 1 group in the new search interface
•	Inverse Scientific Name Hierarchy Display for Material Samples

**Comment by ssbilkhu** [comment]
Restriction component review:
 only display the two fields in read only at the top:
1. CFIA PPC Level 
2. PHAC Containment Level

Parent - Child relations and Restrictions should be treated like Determinations/Collection events unless the child has its own Determination. Need a new ticket for this.

**Comment by ssbilkhu** [comment]
Discussion: need a fields for:
- living/dead
- GMO status
- Patented
- etc (get list from Tara)

Discussion: SeqDB Nagoya Protocol support

**Comment by ron-reade** [comment]
> Discussion: need a fields for:
> 
> * living/dead
> * GMO status
> * Patented
> * etc (get list from Tara)
> 
> Discussion: SeqDB Nagoya Protocol support

It would also be useful for my collection to have a field for Freeze Dried/Live, unless we are planning on separating these another way (like sub collection), but would be nice to see a visual label somewhere anyway!

**Comment by rintoult** [comment]
Here is the website with the PHAC API.
https://open.canada.ca/data/en/dataset/58b91414-dfe8-4d66-aa61-0cfbf19983a7

**Comment by rintoult** [comment]
And here is the PPC list - I took this opportunity to clean up some records that were inappropriately labelled! https://inspection.canada.ca/plant-health/invasive-species/regulated-pests/eng/1363317115207/1363317187811#p

**Comment by cgendreau** [comment]
`GMO`, `Patented` and `Nagoya` should all be entered as tags.

**Comment by rintoult** [comment]
For future versions - can there be a tool that generates "Nagoya" tags based on the country entered? Also an issue for herbarium and CNC i think?

**Comment by rintoult** [comment]
The "Etc list" for consideration:
Codes used by CBS at the Westerdijk Fungal Biodiversity Institute
0-  Open, They actually use a tag/code on records that have no restrictions, this gives a positive indication of status, instead of the absence of restrictions. 
1 - Publication coming, so sharing of material delayed, maximum of two years
2- Quality of strain or batch in question, could be contaminated, not matching identification
3- Dead but some material still in stock
4- Dead Dead Dead - all gone nothing left
5- Budapest Treaty Safe Deposit/Patent - CCFC does not do this
6- MTA, permission restricted this could be also Nagoya restrictions - so some sort of agreement needs to be in place before sharing. 

All of these categories are displayed not only on the database record but also on the label for each material sample.

**Comment by rintoult** [comment]
I had a thought about our conversation in the second part of the meeting on the 31st. 
"existe/n'existe plus"

Could a saved search filter address our second level issue, where we want to separate out those material samples that still exist, no longer exist or are dormant. For Claudia for listing only those that are "alive" could it be a combined search of Date Destroyed is Blank, State is Not Dormant. For me it would be Date Destroyed is Blank, Storage Container is Not Blank.

We still need the higher level Status tag though for Cultured=True, Culture=False Dead Dead Dead - all gone nothing left, Quality of strain or batch in question, could be contaminated or not matching identification,  Dead but some material still in stock

**Comment by rintoult** [comment]
And here is how batches/stock are handled in BioAware
https://www.bio-aware.com/page/VideoStock
Not perfect but one way others deal with it.

**Comment by ron-reade** [comment]
> Restriction component review: only display the two fields in read only at the top:
> 
> 1. CFIA PPC Level
> 2. PHAC Containment Level
> 
> Parent - Child relations and Restrictions should be treated like Determinations/Collection events unless the child has its own Determination. Need a new ticket for this.

This is a tricky one... If the child is say a sub-sample, it may not have the same PPC-1 designation on it... IE... once we extract RNA, that RNA is no longer considered PPC-1 as the original infectious organism was destroyed during the process.
